src/Components/Nav/Nav.css

```diff
@@ -57,3 +45,73 @@
     background-color: unset;
     color: var(--white);
 }
+
+.nav-sort__button {
+    margin-right: 0.5rem;
+}
+
+@media screen and (min-width: 992px) {
+    
+    .nav-dropdown::before {
+        content: '';
+        border-bottom: solid 0.625rem var(--black);
+        border-left: solid 0.625rem transparent;
+        border-right: solid 0.625rem transparent;
+        height: 0.625rem;
+        width: 1.25rem;
+        position: absolute;
+        top: -9px;
+        right: 45px;
+    }
+}
+
+@media (max-width: 992px) {
```

Comment:
Bootstrap have breakpoint variables: https://getbootstrap.com/docs/4.2/getting-started/theming/#breakpoint-variables

Reply:
![image](https://user-images.githubusercontent.com/37185433/52491094-e5f50780-2b8b-11e9-9504-ee6abe6c9e82.png)

not supported